docs/nodes/credentials/Spotify/README.md

```diff
@@ -0,0 +1,19 @@
+# Spotify
```

Comment:
Could you please make sure that the formatting of the document aligns with the updated template?